src/main/java/no/digipost/api/client/ApiServiceMock.java

```diff
@@ -1,429 +0,0 @@
-/**
```

Comment:
Det er en stund siden jeg har vært inne i dette prosjektet, så jeg husker ikke helt hva dette er for noe – men er det meningen å slette denne? 💃 
Den er vel i bruk av noen, eller? :)